tests/virt_autotest/install_package.pm

```diff
@@ -64,6 +65,10 @@ sub install_package {
             assert_script_run("zypper --non-interactive rr dependency_repo");
         }
     }
+    elsif (($repo_0_to_install =~ /SLE-12-SP4/m) && check_var('ARCH', 'aarch64')) {
```

Comment:
elsif may have problem if in future any dependency_repo for 12sp4 of aarch64, suggest to use a separate if directly.

Reply:
Done